.github/workflows/kind-e2e.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
         path: ./src/github.com/tektoncd/chains
 
     - name: Install mirror, kind, knative + sigstore
-      uses: sigstore/scaffolding/actions/setup@main
+      uses: sigstore/scaffolding/actions/setup@74d1900f01c1f717f121e0436c4e1996fa55e986
```

Comment:
think this might need

```
with:
  version: v0.3.0
```

if we're cool with pinning to an older version?

Reply:
Well, I think this might do it (first try ;) )

https://github.com/sigstore/scaffolding/pull/277

That would then allow us to update the chains tests to use proper TUF roots.